utils/lists.py

```diff
@@ -10,6 +10,22 @@
 _all_questions = [
     QuestionAnswer("What is the capital city of Spain?", "Madrid"),
     QuestionAnswer("What is arachnophobia the fear of?", "Spiders"),
+    QuestionAnswer("Stockholm is the capital and largest city of what country?", "Sweden"),
+    QuestionAnswer("When did the website Facebook launch?", "2004"),
+    QuestionAnswer("T/F: The scientific name for the Southern Lights is Aurora Australis?", "T"),
```

Comment:
The user is likely to type "true" or "false" despite the "T/F" signal, so at minimum update both question and answer to use the full words. However, nothing else here is true/false, so perhaps this would be better phrased as something like "What is the scientific name for the Southern Lights?" for the sake of consistency.